### PR TITLE
Remove FunnyTelemetry feature (Resolves #2380)

### LINKED ISSUE #1433: "/funnyguilds funnybin" giving error: Server returned HTTP response code: 502
So, telemetry is broken, I guess...

Full error report:

[here](https://pastebin.com/raw/QZmBZkcV)

### LINKED ISSUE #2380: Remove FunnyTelemetry feature
Hi everyone,

I wanted to bring to your attention that FunnyTelemetry has been down for more than a year now, and it has been causing confusion for new users who think that it is a useful part of our software. However, the reality is that FunnyTelemetry is a broken feature that no one really cares about, and it is holding us down.

As part of our debloating initiative, it would be useful to remove FunnyTelemetry, as it is not adding any value to our software and is only adding to the confusion for new users. This will help us streamline our software and make it more user-friendly.

If anyone has any objections or concerns about this, please feel free to share them. Otherwise, I suggest that we should proceed with removing FunnyTelemetry from our software.

Thank you for your understanding.

Best regards,
Bopke

### PR DESCRIPTION
**Pull request by bopke**
It's so sad that it had to came to this.

Yes, I haven't tried running or even building this now, how did you know?
Gigachad

it also closes #1433, so we can have two in price of one. Great success!

For discussion, please refer to #2380.

Best regards,
Bopke

### CODE REVIEW

File: plugin/src/main/java/net/dzikoysk/funnyguilds/feature/command/user/FunnyGuildsCommand.java [hunk 1]
```diff
@@ -13,7 +13,6 @@
 import net.dzikoysk.funnyguilds.shared.FunnyFormatter;
 import net.dzikoysk.funnyguilds.shared.FunnyTask.AsyncFunnyTask;
 import net.dzikoysk.funnyguilds.shared.TimeUtils;
-import net.dzikoysk.funnyguilds.telemetry.FunnybinAsyncTask;
```
Comment (that-apex) [inline plugin/src/main/java/net/dzikoysk/funnyguilds/feature/command/user/FunnyGuildsCommand.java:16]: No

**Review by that-apex — CHANGES_REQUESTED** [review]
No

File: plugin/src/main/java/net/dzikoysk/funnyguilds/feature/command/user/FunnyGuildsCommand.java (hunk 1 above)
Comment (bopke) [reply plugin/src/main/java/net/dzikoysk/funnyguilds/feature/command/user/FunnyGuildsCommand.java:16]: Hello @that-apex,

I hope this message finds you well. I wanted to bring to your attention a crucial matter that needs to be addressed urgently. As you may be aware, there is a specific line of code that has been causing a lot of issues for our business and is holding us back from moving forward with our project.

After careful analysis and discussion with our development team, we have come to the conclusion that deleting this line of code is imperative for our project to continue successfully. The issue caused by this code is creating a bottleneck in our workflow, leading to delays and errors that are impacting our business and our clients.

We understand that deleting a line of code may seem like a small matter, but it is a critical step towards achieving our business objectives. It is important that we take this action as soon as possible to ensure that our project stays on track.

Gregory, we need your expertise and cooperation to make this happen quickly and efficiently. Your input and support are crucial in this matter, and we would appreciate it if you could work closely with the development team to remove the line of code.

If you have any questions or concerns about this, please feel free to reach out to me directly. Thank you for your understanding and cooperation.

Best regards,
Bopke

**Review by P3ridot — APPROVED** [review]
Future is now

**Review by CDFN — APPROVED** [review]
epic

**Pull request closed without merging**